pymc/step_methods/slicer.py

```diff
@@ -51,61 +52,66 @@ class Slice(ArrayStep):
     name = "slice"
     default_blocked = False
     stats_dtypes_shapes = {
+        "tune": (bool, []),
         "nstep_out": (int, []),
         "nstep_in": (int, []),
     }
 
     def __init__(self, vars=None, w=1.0, tune=True, model=None, iter_limit=np.inf, **kwargs):
-        self.model = modelcontext(model)
-        self.w = w
+        model = modelcontext(model)
+        self.w = np.asarray(w).copy()
```

Comment:
Does `asarray` not make a copy, or is that `array`?

Reply:
asarray returns the same object of it's already a numpy array